accelerator/message.h

```diff
@@ -32,6 +32,10 @@ typedef enum ta_cli_arg_value_e {
   REDIS_HOST_CLI,
   REDIS_PORT_CLI,
 
+  /** DB */
+  DB_ENABLE_CLI,
```

Comment:
I don't think it makes sense to enumerate `DB_ENABLE_CLI`.

Reply:
Without `DB_ENABLE_CLI` ,  Tangle-accelerator will always try to connect to the ScyllaDB host.
There is no existing ScyllaDB host guaranteed. In case TA connects to ScyllaDB host failed, is it OK that shows a warning message or should show an error and terminate TA running?